client/src/components/buttonContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Button = styled.button`
+
+`;
+const ButtonWrapper = styled.div`
+width: 100%
```

Comment:
@danielharmon indent lines 8 - 12 for legibility but other than that everything here looks good.